src/govuk/components/pagination/template.njk

```diff
@@ -0,0 +1,62 @@
+{% set type = params.results.type | default("results") -%}
+
+<nav class="govuk-pagination {{- ' ' + params.classes if params.classes}}" id="pagination-label" aria-label="{{type}} navigation">
+
+  <ul class="govuk-pagination__list">
+    {%- if params.previous %}
+    <li class="govuk-pagination__item govuk-pagination__item--prev">
+      <a class="govuk-pagination__link" href="{{ params.previous.href }}">
+        <span class="govuk-pagination__link-title">
+          <svg class="govuk-pagination__icon" xmlns="http://www.w3.org/2000/svg" height="13" width="17">
+            <path d="m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z"></path>
+          </svg>{%- if params.previous.text %}{{ params.previous.text }}{% else %}Previous
+          <span class="govuk-visually-hidden">page</span>
+          {% endif -%}
+        </span>
+        {%- if params.previous.label %}
+        <span class="govuk-pagination__link-divider govuk-visually-hidden">:</span>
+        <span class="govuk-pagination__link-label">{{ params.previous.label }}</span>
+        {% endif -%}
+      </a>
+    </li>
+    {% endif -%}
+
+    {%- for item in params.items %}
+    {%- if item.type == 'elipses' %}
+    <li class="govuk-pagination__item govuk-pagination__item--elipses">&hellip;</li>
```

Comment:
I think this could be encoded directly, to save a few characters, and make it a bit clearer what it is?

```suggestion
    <li class="govuk-pagination__item govuk-pagination__item--elipses">…</li>
```

Reply:
I understand the main reason to use `&hellip;` is to ensure it works if ever in a non-UTF8 encoded context, though I don't know how common that is these days.